odd-platform-ui/package.json

```diff
@@ -9,12 +9,12 @@
     "build": "pnpm generate && tsc --noEmit && vite build",
     "test": "vitest",
     "tsc": "tsc",
-    "generate": "docker run -v `pwd`/..:/mnt -w /mnt --rm openapitools/openapi-generator-cli:v5.1.1 generate -c odd-platform-ui/openapi-config.yaml",
+    "generate": "chmod u+x ./generate.sh && ./generate.sh",
```

Comment:
Did you try to run `pnpm build` or `gradle build` to make sure that gradle can generate models and build ui too?

Reply:
yes, both `pnpm build` and `gradle build` work